.github/scripts/build_assets/arg_getters.py

```diff
@@ -67,4 +67,15 @@ def get_check_svgs_monthly_args():
     parser.add_argument("icons_folder_path",
                         help="The path to the icons folder",
                         action=PathResolverAction)
-    return parser.parse_args()
\ No newline at end of file
+    return parser.parse_args()
+
+
+def get_release_message_args():
+    """
+    Get the commandline arguments for the check_svgs_monthly.py.
```

Comment:
The comment does not fit the method. It's copy-pasta from `get_check_svgs_monthly_args`. Also the mentioned `check_svgs_monthly.py` does no longer exist, or?

Reply:
Oh right, I just copied it from another function. I'll remove that